webpack.common.js

```diff
@@ -48,30 +48,57 @@ const determineAudienceURL = () => {
   return apiAudienceUrl;
 };
 
+function makeCPApiEndpointURL(currentValue, apiEndpoint) {
```

Comment:
What are these functions for? I don't really follow what they're doing.

Like why do we need to have this logic?

Reply:
This helps derive the CP API URL from the regular API endpoint URL, if the API endpoint set is pointing to an installation, so you wouldn't have to set the CP API URL manually